scripts/run_benchmarking

```diff
@@ -2,93 +2,38 @@
 set -e
 set -x
 
-datapath="/data"
-binpath="/install/bin"
-resultpath="${datapath}/results"
-outpath="${datapath}/out"
-iterations=1
+export datapath="/data"
+export binpath="/install/bin"
+export resultpath="${datapath}/results"
+export outpath="${datapath}/out"
+export iterations=1
 
 # Clean results folder
 mkdir -p "${resultpath}"
 mkdir -p "${outpath}"
 rm "${resultpath}"/* || echo "No results"
 rm "${outpath}"/* || echo "No output"
 
-for test in bbbc mitocheck tubhiswt; do
-    input=unknown
-    case "$test" in
-        bbbc)
-            input=${datapath}/BBBC/NIRHTa-001.ome.tiff
-        ;;
-        mitocheck)
-            input=${datapath}/mitocheck/00001_01.ome.tiff
-        ;;
-        tubhiswt)
-            input=${datapath}/tubhiswt-4D/tubhiswt_C0_TP0.ome.tif
-        ;;
-    esac
-
-    # Java tests
-    (
-        mvn -P metadata -Dtest.iterations=${iterations} -Dtest.input="$input" -Dtest.output=${outpath}/${test}-java.ome.xml -Dtest.results=${resultpath}/${test}-metadata-linux-java.tsv exec:java
-        for i in $(seq ${iterations}); do
-            mvn -P metadata -Dtest.iterations=1 -Dtest.input="$input" -Dtest.output=${outpath}/${test}-java.ome.xml -Dtest.results=${resultpath}/${test}-metadata-linux-java-${i}.tsv exec:java
-        done
-
-        mvn -P pixels -Dtest.iterations=${iterations} -Dtest.input="$input" -Dtest.output=${outpath}/${test}-java.ome.tiff -Dtest.results=${resultpath}/${test}-pixeldata-linux-java.tsv exec:java
-        for i in $(seq ${iterations}); do
-            mvn -P pixels -Dtest.iterations=1 -Dtest.input="$input" -Dtest.output=${outpath}/${test}-java.ome.tiff -Dtest.results=${resultpath}/${test}-pixeldata-linux-java-${i}.tsv exec:java
-        done
-    )
-
-    # C++ tests
-    (
-        ${binpath}/metadata-performance ${iterations} "$input" ${outpath}/${test}-cpp.ome.xml ${resultpath}/${test}-metadata-linux-cpp.tsv
-        for i in $(seq ${iterations}); do
-            ${binpath}/metadata-performance 1 "$input" ${outpath}/${test}-cpp.ome.xml ${resultpath}/${test}-metadata-linux-cpp-${i}.tsv
-        done
-
-        ${binpath}/pixels-performance ${iterations} "$input" ${outpath}/${test}-cpp.ome.tiff ${resultpath}/${test}-pixeldata-linux-cpp.tsv
-        for i in $(seq ${iterations}); do
-            ${binpath}/pixels-performance 1 "$input" ${outpath}/${test}-cpp.ome.tiff ${resultpath}/${test}-pixeldata-linux-cpp-${i}.tsv
-        done
-    )
+run_all_benchmarks=true
+dir="$(dirname "$0")"
+# read the options
+for var in "$@"
+do
+    if [ "$var" == "metadata" ];  then
```

Comment:
From within a Docker container, `./scripts/run_benchmarking tiling` still fails with `unexpected operator`. Since this is a Shell script and not a Bash script, `==` should become `=`.